utils/pom.xml

```diff
@@ -42,19 +41,12 @@
             <version>${slf4j.version}</version>
```

Comment:
might want to remove the version number here to be consistent with the other modules